src/kontrol/options.py

```diff
@@ -434,6 +440,24 @@ def get_argument_type() -> dict[str, Callable]:
             }
         )
 
+    def apply_optimizations(self) -> None:
+        """Applies a series of performance optimizations based on the value of the
+        `optimize_performance` attribute.
+
+        If `optimize_performance` is not `None`, this method will adjust several
+        internal parameters to enhance performance. The integer value of `optimize_performance`
+        is used to set the level of parallelism for frontier exploration and maintenance rate.
+        """
+        if self.optimize_performance is not None:
+            self.assume_defined = True
```

Comment:
Do we want to/can add `--kore-rpc-command='kore-rpc-booster --no-post-exec-simplify '` too?

Reply:
And `--smt-arg smt.ematching=false`

Reply:
Makes sense, thanks!